### DISCUSSION TITLE
1 second bins on the freezing module?

### DISCUSSION THREAD

**Discussion by sarogers9** [DeniseCaiLab/ezTrack#63] [General]
Hi there! If I want the program to output freezing data in 1 second bins for a 45 minute session, is there a better way to do this than manually writing in the frames I need in the summary report module? That would lead to 81,000 lines of code lol!

**Comment by sarogers9** [comment]
never mind - i figured out how to print the freezing variable frame by frame into a csv file :) (sorry, i'm not too familiar with python. thanks!)

**Comment by ZachPenn** [comment]
You could also look into doing a dictionary comprehension for this. The following would provide the bins you want. Just update the fps for your situation.
```
fps = 30
session_secs = 45*60
bin_dict = {sec: (sec*fps, (sec+1)*fps) for sec in np.arange(session_secs)}
```
